2023/2023-ssl-ball-placement-rules.adoc

```diff
@@ -0,0 +1,168 @@
+:source-highlighter: highlightjs
+
+= RoboCup 2022 SSL Ball Placement Technical Challenge Rules
+{docdate}
+:toc:
+:sectnumlevels: 0
+
+// add icons from fontawesome in a up-to-date version
+ifdef::backend-html5[]
+++++
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+++++
+endif::backend-html5[]
+
+:icons: font
+:numbered:
+
+NOTE: References to the male gender in the rules with respect to referees, team
+members, officials, etc. are for simplification and apply to both males and
+females.
+
+== Goals of the Technical Challenge
+
+Given the goal of preparing Division B teams to participate in Division A, this
+technical challenge aims to incentive Division B teams to develop one of the
+skills needed to play a match in Division A. In this case, the challenge is to
+autonomously place the ball using the robots ("*AutoPlacement*").
+
+=== Participation Requirements
+
+All Division B teams are eligible and *must* participate in this challenge.
```

Comment:
I thought we decided against mandatory participation, because we can not *force* teams to do anything after all

Reply:
So, maybe a 'should' is better?

Reply:
'are encouraged' ?